documentation/examples/data-type/rick_roll.ts

```diff
@@ -0,0 +1,14 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+// Returns the infamous YouTube video by Rick Astley as an embed.
+pack.addFormula({
+  name: "Rickroll",
+  description: "Embeds the video \"Never Gonna Give You Up\".",
+  parameters: [],
+  resultType: coda.ValueType.String,
+  codaType: coda.ValueHintType.Embed,
+  execute: async function ([param], context) {
```

Comment:
Never gonna give you up, my unused param.

Reply:
Well played! Fixed.